qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy

```diff
@@ -601,13 +601,13 @@ class AdmissionControllerTest extends BaseSpecification {
         }
     }
 
-    @Retry(attempts = 30, delay = 3)
+    @Retry(attempts = 30, delay = 3 * 1000)
```

Comment:
`@Retry` is not spock's but our custom annotation 

https://github.com/stackrox/stackrox/blob/e54c36d59176870ce957514210d1482a4f41c5c8/qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy#L3

https://github.com/stackrox/stackrox/blob/e54c36d59176870ce957514210d1482a4f41c5c8/qa-tests-backend/annotations/src/main/groovy/io/stackrox/annotations/Retry.groovy#L48-L52